### DISCUSSION TITLE
fatal: Remote branch VERSION not found in upstream origin

### DISCUSSION THREAD

**Discussion by pin32** [CrazyIvan359/mqttany#153] [General]
health to all.
I'm trying to install on raspberry PI4 hosting openhab... but I immediately stopped at this error, the link seems correct and I can open it with a browser. some advice?
link
git clone -b VERSION https://github.com/CrazyIvan359/mqttany.git

**Comment by CrazyIvan359** [comment]
[Right above where you copied that command from](https://crazyivan359.github.io/mqttany/start.html#:~:text=Replace%20VERSION%20with%20the%20version%20you%20want%20to%20download%20(ex%20v0.14.0).) you will find the issue and what do to about it
